rpcs3/Emu/RSX/VK/VKHelpers.h

```diff
@@ -2172,6 +2172,51 @@ namespace vk
 			src.first = false;
 			return VK_SUCCESS;
 		}
+#else
+
+	class swapchain_generic : public native_swapchain_base
```

Comment:
Isn't this supposed to be swapchain_Wayland? Also, this introduces a catch all situation where there is no guarantee that any windowing API would be available on non-windows platforms. One of the two must exist for vulkan to compile, add appropriate guards in Cmake to avoid adding catch-alls all over the code.

Reply:
It is only used on Wayland here, but never actually used it seems.